### PR TITLE
Release/0.78.0

### PR DESCRIPTION
**Pull request by AndyDentFree**
Minor changes to version numbering plus hit all the sample apps to make sure they build with the NuGet.

Also includes final doc changes for 0.78.0

Note pessimistic release date of 2016-09-09 assuming might not get the go-ahead "today".
@fealebenpae please review before I pull this stuff back into master.

### CODE REVIEW

File: Realm.Shared/linq/RealmResultsVisitor.cs [hunk 1]
```diff
@@ -59,6 +59,8 @@ internal static class String
                 internal static readonly LazyMethod StartsWith = Methods.Capture<string>(s => s.StartsWith(""));
 
                 internal static readonly LazyMethod EndsWith = Methods.Capture<string>(s => s.EndsWith(""));
+
+                internal static readonly LazyMethod NullOrEmpty = Methods.Capture<string>(s => string.IsNullOrEmpty(s));
```
Comment (kristiandupont) [inline Realm.Shared/linq/RealmResultsVisitor.cs]: Should be `IsNullOrEmpty`

**Comment by AndyDentFree** [comment]
retest this please

**Comment by AndyDentFree** [comment]
please retest this

**Comment by AndyDentFree** [comment]
retest this please

**Comment by AndyDentFree** [comment]
retest this please

**Comment by kristiandupont** [comment]
:+1:

**Pull request merged**